pandamium_datapack/data/pandamium/functions/triggers/vote_shop.mcfunction

```diff
@@ -1,40 +1,59 @@
+execute if score @s vote_shop matches 1 run tellraw @s [[{"text":"1: ","color":"green"},[{"text":"[1 Diamond]  ","color":"aqua","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -1"},"hoverEvent":{"action":"show_text","contents":[{"text":"3 Vote Credits","color":"green"}]}}]],[{"text":"2: ","color":"green"},[{"text":"[5 Experience Bottles]","color":"aqua","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -2"},"hoverEvent":{"action":"show_text","contents":[{"text":"1 Vote Credit","color":"green"}]}}]]]
+execute if score @s vote_shop matches 1 run tellraw @s [[{"text":"3: ","color":"green"},[{"text":"[1 Golden Apple]  ","color":"aqua","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -3"},"hoverEvent":{"action":"show_text","contents":[{"text":"1 Vote Credit","color":"green"}]}}]],[{"text":"4: ","color":"green"},[{"text":"[1 Endermite Spawn Egg]","color":"aqua","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -4"},"hoverEvent":{"action":"show_text","contents":[{"text":"1 Vote Credit","color":"green"}]}}]]]
+execute if score @s vote_shop matches 1 run tellraw @s [[{"text":"5: ","color":"green"},[{"text":"[1 Mending Book]  ","color":"blue","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -5"},"hoverEvent":{"action":"show_text","contents":[{"text":"25 Vote Credits","color":"green"}]}}]],[{"text":"6: ","color":"green"},[{"text":"[1 Enchanted Golden Apple]","color":"blue","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -6"},"hoverEvent":{"action":"show_text","contents":[{"text":"50 Vote Credits","color":"green"}]}}]]]
+execute if score @s vote_shop matches 1 run tellraw @s [[{"text":"7: ","color":"green"},[{"text":"[A Random Player's Head]  ","color":"gold","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -7"},"hoverEvent":{"action":"show_text","contents":[{"text":"20 Vote Credits","color":"green"}]}}]],[{"text":"8: ","color":"green"},[{"text":"[Your Head]","color":"gold","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -8"},"hoverEvent":{"action":"show_text","contents":[{"text":"100 Vote Credits","color":"green"}]}}]]]
+execute if score @s vote_shop matches 1 run tellraw @s [[{"text":"9: ","color":"green"},[{"text":"[Home Cooldown Reset]  ","color":"yellow","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -9"},"hoverEvent":{"action":"show_text","contents":[{"text":"1 Vote Credit","color":"green"}]}}]],[{"text":"10: ","color":"green"},[{"text":"[TPA Cooldown Reset]","color":"yellow","clickEvent":{"action":"run_command","value":"/trigger vote_shop set -10"},"hoverEvent":{"action":"show_text","contents":[{"text":"1 Vote Credit","color":"green"}]}}]]]
 
 scoreboard players operation @s temp_1 = @s vote_credits
+scoreboard players reset @s temp_2
```

Comment:
I don't think this is necessary? The value is written to before it's read anyways?

Reply:
It isn’t always written to. So in the case that it isn’t written to, the value should be null later. So that line must stay there